napi-inl.h

```diff
@@ -238,13 +238,28 @@ struct AccessorCallbackData {
 // Module registration
 ////////////////////////////////////////////////////////////////////////////////
 
+// Register an add-on based on an initializer function.
 #define NODE_API_MODULE(modname, regfunc)                 \
-  napi_value __napi_ ## regfunc(napi_env env,             \
+  static napi_value __napi_ ## regfunc(napi_env env,             \
```

Comment:
The tests seems to compile/pass without this change. Wondering as to my its in this PR.

Reply:
I added it because I was working on an add-on definition macro right next to it, and we shouldn't be exposing the entry point symbol. You're right though, I'll make it a separate PR.